src/webui/src/components/trial-detail/TableList.tsx

```diff
@@ -295,13 +295,15 @@ class TableList extends React.Component<TableListProps, TableListState> {
                 if (this._isMounted) {
                     // concat default column and finalkeys
                     const item = Object.keys(temp);
-                    const want: Array<string> = [];
-                    Object.keys(item).map(key => {
-                        if (item[key] !== 'default') {
-                            want.push(item[key]);
-                        }
-                    });
-                    showTitle = COLUMN.concat(want);
+                    if (item.length <= 1) {
```

Comment:
could add comments here to explain why check whether item.length is smaller than 1

Reply:
maybe have some questions... I'll test an experiment with dic final keys